R/create_config.R

```diff
@@ -110,9 +120,55 @@ check_output_type_id_datatype <- function(schema_id, output_type_id_datatype) {
   if (pre_v3_1 && not_null) {
     cli::cli_alert_warning(
       "{.arg output_type_id_datatype} not avalaible in
-      schema versions <= {.val v3.0.1}"
+      schema versions < {.val v3.0.1}"
     )
     return(NULL)
   }
   return(output_type_id_datatype)
 }
+
+check_derived_task_ids <- function(derived_task_ids, rounds, model_tasks,
```

Comment:
minor: should this function be moved to its own file since it's being used for `create_round()` as well?

Reply:
Sure. Done in adb57c6